frontend/src/scenes/dashboard/TextCard.tsx

```diff
@@ -0,0 +1,230 @@
+                                                    key={availableColor}
+                                                    active={availableColor === (textTile.color || InsightColor.White)}
+                                                    status="stealth"
+                                                    onClick={() => updateColor(availableColor)}
+                                                    data-attr={`set-text-tile-ribbon-color-${availableColor}`}
+                                                    icon={
+                                                        availableColor !== InsightColor.White ? (
+                                                            <Splotch color={availableColor as string as SplotchColor} />
+                                                        ) : null
+                                                    }
+                                                    fullWidth
+                                                >
+                                                    {availableColor !== InsightColor.White
+                                                        ? capitalizeFirstLetter(availableColor)
+                                                        : 'No color'}
+                                                </LemonButton>
+                                            )),
+                                            placement: 'right-start',
+                                            fallbackPlacements: ['left-start'],
+                                            actionable: true,
+                                        }}
+                                        fullWidth
+                                    >
+                                        Set color
+                                    </LemonButtonWithPopup>
+                                )}
+                                <LemonDivider />
+                                {removeFromDashboard && (
+                                    <LemonButton
+                                        status="danger"
+                                        onClick={removeFromDashboard}
+                                        fullWidth
+                                        data-attr="remove-text-tile-from-dashboard"
+                                    >
+                                        Remove from dashboard
+                                    </LemonButton>
+                                )}
+                            </>
+                        }
+                    />
+                </div>
+            </div>
+            <LemonDivider />
+            <div className="flex p-2 pl-4">
+                {textTile.color &&
+                    textTile.color !==
+                        InsightColor.White /* White has historically meant no color synonymously to null */ && (
+                        <div className={clsx('DashboardCard__ribbon ml-2', textTile.color)} />
```

Comment:
This doesn't looks right. The color ribbon should be consistently in the "meta" part of the card (above ridge) rather than in the "content" part (below ridge).

<img width="1492" alt="Screen Shot 2022-09-22 at 14 45 47" src="https://user-images.githubusercontent.com/4550621/191750702-adc5d13e-02c6-40eb-8559-982aa29efbce.png">

Perhaps we could just omit the color feature from text tiles until we figure out what it should really look like? Honestly I don't think there's a ton of value in this particular feature.

Besides that, the UI of text cards and the Markdown modal seems really rough, but I'm totally okay with dedicating a separate PR to them, as those problems are independent of existing code.